agent/src/bpf/audit.bpf.c

```diff
@@ -31,12 +35,18 @@ record_new_context (struct pt_regs *ctx)
   long context;
   err = bpf_usdt_arg (ctx, 0, &context);
   if (err < 0)
-    return err;
+    {
+      DEBUG ("unable to get the first argument: %ld\n", err);
+      return err;
+    }
```

Comment:
Would it make sense to do this type of stuff in a function to avoid some code duplication? I get it that on the other hand, we would have to check the return code anyway, but maybe if we want to add some logic or change the message in future? :thinking:

Reply:
I changed this to leverage the varargs of BPF_USDT, so checking for each bpf_usdt_arg is not necessary.